### PR TITLE
🐛Fix amp-inputmask attributes stripping in amp-mustache 0.2. 

### PR DESCRIPTION
**Pull request by cvializ**
And update example

@choumx was there any reason why the purifier didn't already allow `value`?

### CODE REVIEW

File: src/purifier.js [hunk 1]
```diff
@@ -162,6 +162,11 @@ export const WHITELISTED_ATTRS_BY_TAGS = {
     'custom-validation-reporting',
     'target',
   ],
+  'input': [
+    'value',
+    'mask-output',
+    'mask',
+  ],
```
Comment (choumx) [inline src/purifier.js]: `value` and `mask` should already be allowed: https://github.com/cure53/DOMPurify/blob/master/src/attrs.js

Try it out by adding a simple test case in test-purifier.js.

Comment (cvializ) [reply src/purifier.js]: `mask` looks like it's only allowed for svg, and this is for an input tag. I'll check the error messages to see what the issue with `value` was.

Comment (choumx) [reply src/purifier.js]: DOMPurify doesn't use tag-specific whitelists for attributes. We should add a unit test for this change in test-purifier.js anyways.

Comment (cvializ) [reply src/purifier.js]: Done. PTAL

Comment (cvializ) [reply src/purifier.js]: Done. PTAL

**Pull request merged**